util.go

```diff
@@ -18,16 +18,17 @@ import (
 type contextKey string
 
 const (
-	multiStatementCount   contextKey = "MULTI_STATEMENT_COUNT"
-	asyncMode             contextKey = "ASYNC_MODE_QUERY"
-	queryIDChannel        contextKey = "QUERY_ID_CHANNEL"
-	snowflakeRequestIDKey contextKey = "SNOWFLAKE_REQUEST_ID"
-	fetchResultByID       contextKey = "SF_FETCH_RESULT_BY_ID"
-	fileStreamFile        contextKey = "STREAMING_PUT_FILE"
-	fileTransferOptions   contextKey = "FILE_TRANSFER_OPTIONS"
-	enableHigherPrecision contextKey = "ENABLE_HIGHER_PRECISION"
-	arrowBatches          contextKey = "ARROW_BATCHES"
-	arrowAlloc            contextKey = "ARROW_ALLOC"
+	multiStatementCount     contextKey = "MULTI_STATEMENT_COUNT"
+	asyncMode               contextKey = "ASYNC_MODE_QUERY"
+	queryIDChannel          contextKey = "QUERY_ID_CHANNEL"
+	snowflakeRequestIDKey   contextKey = "SNOWFLAKE_REQUEST_ID"
+	fetchResultByID         contextKey = "SF_FETCH_RESULT_BY_ID"
+	fileStreamFile          contextKey = "STREAMING_PUT_FILE"
+	fileTransferOptions     contextKey = "FILE_TRANSFER_OPTIONS"
+	enableHigherPrecision   contextKey = "ENABLE_HIGHER_PRECISION"
+	arrowBatches            contextKey = "ARROW_BATCHES"
+	arrowAlloc              contextKey = "ARROW_ALLOC"
+	enableOriginalTimestamp contextKey = "ENABLE_ORIGINAL_TIMESTAMP"
```

Comment:
PLease extend readme with description of the new paramter

Reply:
There is no such section in the readme file. I'll later create the ticket for the doc team to update https://pkg.go.dev/github.com/snowflakedb/gosnowflake#WithArrowBatches

Reply:
@sfc-gh-dstempniak for Go, we don't use (yet) docs.snowflake.com so please feel free to directly update `doc.go` so it can be generated automatically at pkg.go.dev

Reply:
The parameter (or rather the function enabling the functionality) is described in `util.go`. So as I understand it won't go to [README](https://pkg.go.dev/github.com/snowflakedb/gosnowflake#section-readme) section, but it will go to [Documentation](https://pkg.go.dev/github.com/snowflakedb/gosnowflake#pkg-functions) section like other similar functions. @sfc-gh-dszmolka  Am I right? 
If so, I think it is enough.